source/Halibut.Tests/Support/SerilogLoggerBuilder.cs

```diff
@@ -105,5 +115,35 @@ public void Emit(LogEvent logEvent)
                 }
             }
         }
+
+        public class TraceLogsForFailedTestsSink : ILogEventSink
+        {
+            private readonly MessageTemplateTextFormatter _formatter;
+
+            public TraceLogsForFailedTestsSink(MessageTemplateTextFormatter formatter) => _formatter = formatter != null ? formatter : throw new ArgumentNullException(nameof(formatter));
```

Comment:
Do we need to guard against nulls? We tend not to do that at Octopus, and rely instead of compile time checking.